### ISSUE TITLE
pybindings - don't auto-build dependencies (╯°□°)╯︵ ┻━┻ (Trac #1032)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1026]
this is fine with `make` or `make all` but when trying to do a partial/incremental build, you have to keep failing python scripts to see what pybindings you need (without looking into CMakeLists.txt)

Migrated from https://code.icecube.wisc.edu/ticket/1032
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:01:19", 
    "description": "this is fine with `make` or `make all` but when trying to do a partial/incremental build, you have to keep failing python scripts to see what pybindings you need (without looking into CMakeLists.txt)", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251279109761", 
    "component": "infrastructure", 
    "summary": "pybindings - don't auto-build dependencies (\u256f\u00b0\u25a1\u00b0)\u256f\ufe35 \u253b\u2501\u253b", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-06-25T19:00:21", 
    "milestone": "", 
    "owner": "nega", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150810T23:46:20: 
* olivas changed owner from "" to "nega"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150909T19:44:40: 
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 2300:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2300"
> setup dependencies for pybindings
> 
> a bit overkill i think, but link-time and run-time dependencies should
> now auto-build when selecting an individual project. For example, `make
> tpx-pybindings` will build tpx, its dependencies, and all pybindings needed
> to `import tpx` in a python program. (Assuming they were defined correctly
> in `i3_add_pybindings()`.
> 
> closes #1026
> ```


* nega changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:01:19: nwhitehorn commented:

> In changeset 170115:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170115"
> setup dependencies for pybindings
> 
> a bit overkill i think, but link-time and run-time dependencies should
> now auto-build when selecting an individual project. For example, `make
> tpx-pybindings` will build tpx, its dependencies, and all pybindings needed
> to `import tpx` in a python program. (Assuming they were defined correctly
> in `i3_add_pybindings()`.
> 
> closes #1026
> ```
